adam/src/main/kotlin/com/malinskiy/adam/request/sync/PushRequest.kt

```diff
@@ -171,13 +171,13 @@ class PushRequest(
         serial: String?,
         dirsToCreate: List<String>
     ) {
-        execute(ShellCommandRequest("mkdir -p $destination"), serial)
+        execute(ShellCommandRequest("mkdir -p ${escape(destination)}"), serial)
```

Comment:
Do you mind using `bashEscape` that is available as an extension on `List<String>`? I don't want to reimplement escaping per request